bottomline_recordandreplay/assets/security/rule-Bottomline__Customer_Maintenance_on_VIP.json

```
@@ -0,0 +1,35 @@
+{
+  "name": "Bottomline: Customer Maintenance on VIP",
+  "isEnabled": true,
```

```suggestion
  "name": "Customer Maintenance on VIP",
  "partnerRuleId":"ext-001-btm",
  "isEnabled": false,
```
Added a missing field and removed the integration name from the rule name since that will already display automatically.